src/web/components/elements/InteractiveBlockComponent.tsx

```diff
@@ -57,18 +63,171 @@ and is sent with all interactive elements in the scriptUrl from CAPI to do a two
 It has not been updated since 2016.
 
 MIGRATION FROM FRONTEND
-- TODO For the standard boot.js, we will re-write the behavior in modern JS to avoid the requirement of an AMD loader
+- For the standard boot.js, we have re-written the behavior in modern JS to avoid the requirement of an AMD loader
 and to avoid loading the boot.js file.
 - For all other files that do not load the standard boot.js, we'll add a AMD loader to the page
 
 For the remaining few we dynamically load and AMD loader and support the contract as defined with curl AMD loader
 
 */
+const supportingMinHeight = 200;
+const inlineMinHeight = 500;
+const getMinHeight = (role: string, loaded: boolean) => {
+	if (loaded) {
+		return `auto`;
+	}
+	return role === 'supporting'
+		? `${supportingMinHeight}px`
+		: `${inlineMinHeight}px`;
+};
```

Comment:
This is maybe just a naming thing but the default here is called 'inline...' but it's also capturing all the other roles

Reply:
I see this ternary is used in a few places. What about having a `decideHeight(role: RooleType)` function with a simple switch to better convey intentions and defaults?